src/Xamarin.Android.Build.Tasks/Utilities/Aapt2Daemon.cs

```diff
@@ -271,9 +271,13 @@ private void Aapt2DaemonStart ()
 			{
 				// Ignore this error. It occurs when the Task is cancelled.
 			}
-			aapt2.StandardInput.WriteLine ("quit");
-			aapt2.StandardInput.WriteLine ();
-			aapt2.WaitForExit ((int)TimeSpan.FromSeconds (5).TotalMilliseconds);
+			try {
+				aapt2.StandardInput.WriteLine ("quit");
+				aapt2.StandardInput.WriteLine ();
+				aapt2.WaitForExit ((int)TimeSpan.FromSeconds (5).TotalMilliseconds);
+			} catch (IOException) {
+				// Ignore this error. It occurs when the Build it cancelled.
```

Comment:
Should we log this error, "just in case", perhaps as a Diagnostic message?

Reply:
We do not have access to the MSBuild Logger from the daemon. I think the best we could do is a Console.WriteLine, unless I refactor this a bit.  

This only occurs when the user actively cancels the build btw. But I see your point about loggin this just in case.